sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <template v-if="applicationStatus !== ApplicationStatus.cancelled">
+    <body-header title="Track your application" />
+    <v-slider
+      v-model="trackerApplicationStatus"
+      :ticks="applicationStatusTracker"
+      :max="4"
+      step="1"
+      show-ticks="always"
+      tick-size="0"
+      track-color="readonly"
+      :track-fill-color="trackFillColor"
+      :thumb-size="thumbSize"
+      :thumb-color="thumbColor"
+      track-size="20"
+      readonly
+      :disabled="disabled"
+      class="application-slider"
+    >
+      <template #tick-label="{ tick, index }">
+        <span
+          v-if="index === trackerApplicationStatus"
+          class="label-bold default-color"
+          >{{ tick.label }}
+          <v-icon
+            v-if="hasDeclinedCard"
+            icon="fa:fas fa-exclamation-circle"
+            :size="20"
+            color="danger"
+            class="pl-4"
+        /></span>
+        <span class="label-value default-color" v-else>{{ tick.label }} </span>
+      </template>
+    </v-slider>
+
+    <draft
```

Comment:
I would prefer that the show/hide of all these components would be based on a simple status of the `ApplicationStatus` in a way that is is not dependent on `applicationStatusTracker[trackerApplicationStatus]`.